### PR TITLE
New branch added - fonts added - some scss files created

### PR DESCRIPTION
**Pull request by mistrs2p**
Welcome @a3dho3yn

### CODE REVIEW

File: app/Http/Controllers/PostsController.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    // index => listing of all the posts
```
Comment (a3dho3yn) [inline app/Http/Controllers/PostsController.php:17]: These comments are redundant. What's the problem with redundancy? The cost of keeping them up to date, without any benefits.

File: app/Http/Controllers/PostsController.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    // store => submit to the function , take care of submitting it to the model to the database
+    // edit => edit form
+    // update
+    // delete
+    // show => showing a single post
+
+    public function index()
+    {
+        //fetch all of the data in the model in the table
+        // $posts = Post::all(); // return this line and stops. not loading the view
```
Comment (a3dho3yn) [inline app/Http/Controllers/PostsController.php:28]: There's no reason to comment codes. If it's not valid anymore, you can remove it (and let VCS keep the history)

File: app/Post.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    // Table Name
+    protected $table = 'posts';
+    // Primary Key
+    public $primaryKey = 'id';
+    // Timestamps
```
Comment (a3dho3yn) [inline app/Post.php]: What additional information these comments give us?

Comment (mistrs2p) [reply app/Post.php]: we can add these information to set our table name or to set our primary key  in our order

File: app/Http/Controllers/PostsController.php (hunk 2 above)
Comment (mistrs2p) [reply app/Http/Controllers/PostsController.php:28]: It helps me to memorize different format of doing One thing in purpose. Is it wrong?

File: app/Http/Controllers/PostsController.php (hunk 1 above)
Comment (mistrs2p) [reply app/Http/Controllers/PostsController.php:17]: Ok. I figure that out

**Pull request closed without merging**